### PR TITLE
[WIP] Zapify command

### LINKED ISSUE #417: Zap Command
http://vemdezapbe.be

### PR DESCRIPTION
**Pull request by pedrofracassi**
Closes #417

### CODE REVIEW

**Comment by metehus** [comment]
what needs to close this? @pedrofracassi

File: src/commands/memes/zapify.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        .setTitle(t('commands:zapify.chooseHumor'))
+        .setDescription([
+          `**${t('commands:zapify.pleaseReact', { COLLECTOR_TIMEOUT })}**`,
+          Object.keys(humorEmoji).map(h => {
+            return `${humorEmoji[h]} ${t(`commands:zapify.humors.${h}`)}`
+          }).join('\n')
+        ].join('\n\n'))
+    )
+
+    for (const e of Object.keys(humorEmoji)) {
```
Comment (metehus) [inline src/commands/memes/zapify.js]: WHY USING FORRR

Comment (metehus) [reply src/commands/memes/zapify.js:40]: nvm

**Comment by pedrofracassi** [comment]
Stale!

**Comment by Doges** [comment]
ae caralho

**Pull request closed without merging**